src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/Controllers/Domain/PlayersController.cs

```diff
@@ -291,27 +296,30 @@ public JsonResult GetFreePlayers(string searchString, string excludeList, string
             }
 
             var result = query.OrderBy(p => p.LastName)
-                              .ToList()
-                              .Select(p => PlayerNameViewModel.Map(p));
+#pragma warning disable S2971 // "IEnumerable" LINQs should be simplified Enity franework error must be ToList()
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not suppress issues. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1309)